gravity-forms/gw-none-of-the-above-checkbox.js

```diff
@@ -20,11 +20,19 @@
  *    any Checkbox field to which this should be applied.
  */
 $( '.gw-none-of-the-above' ).each( function() {
-	
+
 	var $field  = $( this );
 	var $last   = $field.find( '.gchoice:last-child input' );
 	var $others = $field.find( 'input' ).not( $last );
 
+	// If "None of the Above" choice is checked by default.
+	if ( $last[0].checked ) {
```

Comment:
FWIW, `$(last).prop('checked')` would be safer in case `$last` is somehow empty. Not a big deal since this is in a snippet, but if it was in a perk, I'd probably change it.